recipes/hana/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class HanaConan(ConanFile):
+    name = "hana"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://boostorg.github.io/hana/"
+    description = "Hana is a header-only library for C++ metaprogramming suited for computations on both types and values."
+    license = "BSL-1.0"
+    topics = ("hana", "metaprogramming", "boost")
+    settings = "compiler"
+    no_copy_source = True
+    exports_sources = "CMakeLists.txt"
+
+    _compiler_cpp14_support = {
+        "gcc": "4.9.3",
+        "Visual Studio": "14.0",
+        "clang": "3.4",
+        "apple-clang": "3.4",
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "_source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("hana-" + self.version, self._source_subfolder)
+
+    def config_options(self):
```

Comment:
```suggestion
    def configure(self):
```

config options is only used for removing options. Otherwise, use `configure`